packages/docs-site/src/content/docs/network-reference/network-configuration.md

```diff
@@ -3,9 +3,30 @@ title: Network configuration
 description: The network configuration page describes many of the configuration details of the network.
 ---
 
+## Cooldown Window, Proving Window and Bonds
+
+This segment lists the cooldown windows and proving windows for every tier of proof in the Taiko protocol.
+
+The cooldown window describes how long after the proof has been provided that the block is verified.
+
+The proving window describes how long a prover has to provide a proof for a block after they have been assigned the block (i.e. a block has been proposed with them as the assigned prover).
+
+We've appended a 4 hour "grace period" temporarily to proving windows, the reason is described [here](https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/contracts/layer1/tiers/TierProviderBase.sol#L13). You can consider the below proving windows + 4 hours as the actual proving window until further notice!
+
+You can find all the corresponding values below in the above link too!
+
+| Tier              | Cooldown Window | Proving Window | Bond      |
+| ----------------- | --------------- | -------------- | --------- |
+| Optimistic        | 1440 minutes    | 15 minutes     | 75 TAIKO  |
+| SGX               | 240 minutes     | 60 minutes     | 150 TAIKO |
+| ZK                | 240 minutes     | 180 minutes    | 225 TAIKO |
+| SGX & ZK          | 240 minutes     | 180 minutes    | 225 TAIKO |
+| Guardian Minority | 240 minutes     | 2880 minutes   | 225 TAIKO |
```

Comment:
seems proving window for gp in code is just `GRACE_PERIOD`?

Reply:
@RogerLamTd please do not use the code on the main branch, you should fine the deployed contract URL and use the verified code as reference.

Reply:
grace period is also removed.

Reply:
@dantaik I see grace period has yet to be removed on the live contract, we're leaving it for a future upgrade if I'm not mistaken?